package.json

```diff
@@ -2,13 +2,25 @@
   "name": "rollup-plugin-livereload",
   "version": "2.0.5",
   "description": "Rollup plugin for LiveReload that watches the bundle and reloads the page on change",
-  "main": "dist/index.cjs.js",
-  "module": "dist/index.es.js",
-  "jsnext:main": "dist/index.es.js",
-  "types": "index.d.ts",
+  "type": "module",
+  "main": "dist/index.cjs",
+  "module": "dist/index.js",
+  "types": "dist/index.d.cts",
+  "exports": {
```

Comment:
The `exports` field has been added so that we can define separate entry points for ESM and CJS, and also for the respective types.